tests/components/cover/test_command_line.py

```diff
@@ -44,41 +42,42 @@ def test_query_state_value(self):
 
     def test_state_value(self):
         """Test with state value."""
-        with tempfile.TemporaryDirectory() as tempdirname:
-            path = os.path.join(tempdirname, 'cover_status')
-            test_cover = {
-                'command_state': 'cat {}'.format(path),
-                'command_open': 'echo 1 > {}'.format(path),
-                'command_close': 'echo 1 > {}'.format(path),
-                'command_stop': 'echo 0 > {}'.format(path),
-                'value_template': '{{ value }}'
-            }
-            self.assertTrue(setup_component(self.hass, cover.DOMAIN, {
-                'cover': {
-                    'platform': 'command_line',
-                    'covers': {
-                        'test': test_cover
-                    }
+        test_cover = {
+            'command_state': 'cat cover_status',
+            'command_open': 'echo 1 > cover_status',
+            'command_close': 'echo 1 > cover_status',
+            'command_stop': 'echo 0 > cover_status',
+            'value_template': '{{ value }}'
+        }
+        self.assertTrue(setup_component(self.hass, cover.DOMAIN, {
```

Comment:
Ok.  I'll start a PR now so that we are at least tracking the issue and note the files that I think need to be changed.  Then once this PR and your `device_tracker` PR are complete, I'll rebase those changes into the PR and continue work on it.

If it's ok with you, I'll fix this instance of `assertTrue` in the new PR, since this `assertTrue` call was carried over from the existing code.  But, if you want me to fix it in this PR, I can do that, too.  Whichever you prefer.